cypress/e2e/error-handling.cy.js

```diff
@@ -0,0 +1,43 @@
+describe('template spec', () => {
+  let npsApiKey = Cypress.env('nps_api_key');
+
+  it('Displays a message for server-side errors when network request is not successful', () => {
+    cy.intercept({
+      pathname: '/api/v1/parks',
+      query: {
+        limit: '469',
+        api_key: npsApiKey,
+      },
+    }, {
+      statusCode: 500,
+      body: {
+        message: 'Internal Server Error'
```

Comment:
Checking for proper status codes, this looks great and a clear message of what could be going on.